src/commands/infra/generate.ts

```diff
@@ -32,24 +33,13 @@ export enum DefinitionYAMLExistence {
   PARENT_ONLY
 }
 
-export const fetchValues = (opts: ICommandOptions): string => {
-  if (opts.project) {
-    logger.info(`spk will generate the following project: ${opts.project}`);
-    return opts.project;
-  }
-  logger.warn(
-    `No project folder was provided, spk will generate the current folder as a project`
-  );
-  return process.cwd();
-};
-
 export const execute = async (
   opts: ICommandOptions,
   exitFn: (status: number) => Promise<void>
 ) => {
-  const projectPath = fetchValues(opts);
   const parentPath = process.cwd();
-
+  const projectPath = opts.project || "";
```

Comment:
i think it should be 
`const projectPath = opts.project || parentPath;`

Reply:
at one point, we check if projectPath and parentPath are the same or not. https://github.com/CatalystCode/spk/blob/master/src/commands/infra/generate.ts#L356

Reply:
Fixed!